packages/server/README.md

```diff
@@ -44,39 +49,55 @@ POSTGRES_PORT=5432
 POSTGRES_HOST=localhost
 ```
 
-Make sure that the provided database and user exist as the server will not create them. When running on macOS or Windows through Docker Desktop, a mapping of localhost is made automatically. On Linux, you can add `--net=host --add-host=host.docker.internal:127.0.0.1` to the `docker run` command line to make the mapping happen. Any other `POSTGRES_HOST` than localhost or 127.0.0.1 should work as expected without further action.
+Ensure that the provided database and user exist as Joplin server will not create them. When running on macOS or Windows through Docker Desktop, a mapping of localhost is made automatically. On Linux, you can add `--net=host --add-host=host.docker.internal:127.0.0.1` to the `docker run` command line to make the mapping happen. Any other `POSTGRES_HOST` than localhost or 127.0.0.1 should work as expected without further action.
 
-### Using docker-compose
+### Using Docker-Compose
 
-A [sample docker-compose file](https://github.com/laurent22/joplin/blob/dev/docker-compose.server.yml
- ) is available to show how to use Docker to install both the database and server and connect them:
+1. Using the sample [docker-compose file](https://github.com/laurent22/joplin/blob/dev/docker-compose.server.yml), create a docker compose file in the location of your Docker configuration files. Example: /home/[user]/docker/docker-compose.yml
+2. Update the following fields:
 
-## Setup reverse proxy
 
-Once Joplin Server is running, you will then need to expose it to the internet by setting up a reverse proxy, and that will depend on how your server is currently configured, and whether you already have Nginx or Apache running:
+```conf
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+POSTGRES_USER=${POSTGRES_USER}
+APP_BASE_URL=${APP_BASE_URL}
+```
+
+
+- `APP_BASE_URL`: This is the base public URL where the service will be running. 
+	- If Joplin Server needs to be accessible over the internet, configure `APP_BASE_URL` as follows: `https://example.com/joplin`. 
+	- If Joplin Server does not need to be accessible over the internet, set the the `APP_BASE_URL` to your server's hostname. For Example: 		 `http://[hostname]:22300`. The base URL can include the port.
+- `APP_PORT`: The local port on which the Docker container will listen. 
+	- You would typically map this port to 443 (TLS) with a reverse proxy.
+	- If Joplin Server does not need to be accessible over the internet, the port can be mapped to 22300.
+
+
+## Setup Reverse Proxy (Optional)
```

Comment:
Please don't put "(optional)" in titles. Those may be linked to, and it makes the link format awkward.